app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate(params[:session][:password])
```

Comment:
`&.` nghĩa là gì em

Reply:
nguyên nó là ntn: if user && user.authenticate(params[:session][:password])
nhưng rubocop báo lỗi Use safe navigation (&.) instead of checking if an object exists before calling the method.
toán tử &. nghĩa là  điều hướng an toàn. kiểm tra điều kiện tồn tại trước khi thực hiện hành động.

Reply:
ukm, cái này chị biết, chị hỏi em hiểu thế nào thôi
Đọc thêm link: http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/